specs/commandline/policy.md

```diff
@@ -0,0 +1,696 @@
+            "signatureVerification": {                                  // The level of verification - strict, permissive, audit, skip
+                "level": "strict"
+                "override" : {
+                     "expiry" : "log",
+                     "authenticity": "log"
+                }
+            },
+            "trustStores": [ "ca:wabbit-networks-dev" ],                // The trust stores that contains the X.509 certificates
+            "trustedIdentities": [                                      // Identities that are trusted to sign the artifact.
+                "x509.subject: C=US, ST=WA, L=Seattle, O=Example, OU=Dev, CN=wabbit-networks.io"
+            ]
+        },
+        {
+            "name": "wabbit-networks-prod",                             // Name of the second policy statement.
+            "registryScopes": [ "prod.wabbitnetworks.io/net-monitor" ],       
+            "signatureVerification": {                                
+                "level": "permissive"
+            },
+            "trustStores": [ "ca:wabbit-networks-prod" ],                  
+            "trustedIdentities": [                                    
+                "x509.subject: C=US, ST=WA, L=Seattle, O=Example, OU=Prod, CN=wabbit-networks.io"
+            ]
+        }
+    ]
+}
+```
+
+## Outline
+
+### notation policy command
+
+```text
+Manage trust policies for signature verification.
+
+Usage:
+  notation policy [command]
+
+Available Commands:
+  add           Add trust policies
+  delete        Delete trust policies
+  list          List trust policies
+  update        Update trust policies
+
+Flags:
+  -h, --help   help for policy
+```
+
+### notation policy add
+
+```text
+Add trust policies.
+
+Usage:
+  notation policy add (--file <policy_file_path> | --name <policy_name>) [flags]
+
+Flags:
+      --custom-level       stringArray   {key}={value} pairs that represent a custom level to override existing verification level
+  -f  --file               string        path to a trust policy file in JSON format
+  -n  --name               string        name of the trust policy statement
+  -h, --help                             help for add
+      --scope              stringArray   repository URIs to which the policy applies (default ["*"])
+      --trust-store        stringArray   trust stores in format "<trust_store_type>:<trust_store_name>"
+      --verification-level string        verification level, options: "strict", "permissive", "audit", "skip" (default "strict")
+      --x509-cert          stringArray   paths to x509 certificate file that certificate subject is retrieved from
+      --x509-id            stringArray   trust identities, user trusted x509 certificate subjects
+```
+
+### notation policy delete
+
+```text
+Delete trust policies. User cannot delete all the trust policies, at least one trust policy should be configured for signature verification.
```

Comment:
`user cannot delete all the trust policies, at least one trust policy should be configured for signature verification.`
Instead of restricting it should add a disclaimer that for signature verification, notation needs at least one trust-policy?